setup.py

```diff
@@ -50,5 +50,4 @@
         "Release Notes": "https://8bitscoding.io/2020/08/30/pygamelib-v1-2-0-release-notes/",  # noqa: E501
     },
     python_requires=">=3.6",
-    obsolotes="hac-game-lib",
```

Comment:
At first, I didn't mind that one, but after more considerations, I think we should keep that. There is no garantuee that people made the switch in just one major release. We should keep the depreciation at least until v1.5.0.

Reply:
there is a typo there. "obsolotes". that is correct?

Reply:
absolutely... There's a typo... :facepalm: